wear/src/main/java/io/homeassistant/companion/android/util/CommonFunctions.kt

```diff
@@ -0,0 +1,73 @@
+package io.homeassistant.companion.android.util
+
+import android.content.Context
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.colorResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.unit.sp
+import androidx.wear.compose.material.ChipColors
+import androidx.wear.compose.material.ChipDefaults
+import androidx.wear.compose.material.Text
+import com.mikepenz.iconics.IconicsDrawable
+import com.mikepenz.iconics.typeface.IIcon
+import com.mikepenz.iconics.typeface.library.community.material.CommunityMaterial
+import io.homeassistant.companion.android.R
+import io.homeassistant.companion.android.home.HomePresenter
+import io.homeassistant.companion.android.viewModels.EntityViewModel
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.Job
+import kotlinx.coroutines.launch
+
+class CommonFunctions {
+
+    private val mainScope: CoroutineScope = CoroutineScope(Dispatchers.Main + Job())
+
+    @Composable
+    fun SetTitle(id: Int) {
+        Text(
+            text = stringResource(id = id),
+            textAlign = TextAlign.Center,
+            fontSize = 15.sp,
+            fontWeight = FontWeight.Bold,
+            modifier = Modifier
+                .fillMaxWidth()
+        )
+    }
+
+    @Composable
+    fun setChipDefaults(): ChipColors {
+        return ChipDefaults.primaryChipColors(
+            backgroundColor = colorResource(id = R.color.colorAccent),
+            contentColor = Color.Black
+        )
+    }
+
+    fun updateFavorites(entityViewModel: EntityViewModel, presenter: HomePresenter) {
+        mainScope.launch { entityViewModel.favoriteEntities = presenter.getWearHomeFavorites().toMutableSet() }
+    }
+
+    fun saveFavorites(favorites: Set<String>, presenter: HomePresenter) {
+        mainScope.launch { presenter.setWearHomeFavorites(favorites.toSet()) }
+    }

```

Comment:
Pass in the scope so you can make these top level.

Reply:
sure, thank you :)